src/v/cloud_storage/inventory/inv_consumer.cc

```diff
@@ -15,7 +15,9 @@
 #include "bytes/iostream.h"
 #include "cloud_storage/logger.h"
 #include "hashing/xx.h"
-#include "serde/serde.h"
```

Comment:
removing these kitchen sink `serde/serde.h` header uses really causes a lot of fall out, as seen in this commit.